test/integration/scratch-tests.js

```diff
@@ -0,0 +1,92 @@
+/* global vm, Promise */
+const {Chromeless} = require('chromeless');
+const test = require('tap').test;
+const path = require('path');
+const fs = require('fs');
+const chromeless = new Chromeless();
+
+const testFile = (file) => {
```

Comment:
Unnecessary parens around single argument in arrow function.

Reply:
if this is a rule we like to follow, can we enable `"arrow-parens": ["error", "as-needed"]` in the eslint rules?

Reply:
it seems this rule is enabled in the `scratch/es6` ruleset which isn't used in this repo - enabling that...